web/source/kmwbase.js

```diff
@@ -1084,14 +1084,18 @@ if(!window['keyman']['loaded']) {
      * @param       {Event}      e        event
      * @return      {Object}              HTML element
      */       
-    util.eventTarget = function(e)
-    {
-      if(e && e.target)        // most browsers
+    util.eventTarget = function(e) {
+      if(!e) {
+        return null;
+      } else if (e.target) {       // most browsers
         return e.target;
-      else if(window.event)     //IE 8 (and earlier)
+      } else if (e.srcElement) {
+        return e.srcElement;
+      } else if(window.event) { //IE 8 (and earlier)
         return window.event.srcElement;
```

Comment:
Note that if `keymanweb._GetEventObject(e);` is called before this method is called, this is **not** an overall change of behavior.  That method routes `window.element` to our input `e` before this method is called, so it's actually a dead branch in that case.